packaging/functions.sh

```diff
@@ -9,7 +9,7 @@
 # Compile and publish (using Mono) the core engine and specified mod assemblies to the target directory
 # Arguments:
 #   SRC_PATH: Path to the root OpenRA directory
-#   DEST_PATH: Path to the root of the install destination (will be created if necessary)
```

Comment:
I didn't mean to change these internal packaging functions. DEST_PATH and the comment is correct for these.

Reply:
Reverted it.